packages/clarity-js/src/data/metadata.ts

```diff
@@ -99,15 +109,19 @@ function num(string: string, base: number = 36): number {
 }
 
 function user(): number {
-  let id;
+  let id = cookie(Constant.STORAGE_KEY);
```

Comment:
Shouldn't this be "USERID_KEY" and not STORAGE_KEY?

Reply:
Currently they are same constants. I will duplicate to avoid confusion.

Reply:
No change made yet that I saw in iteration #6.

---
In reply to: [467635724](https://github.com/microsoft/clarity/pull/31#discussion_r467635724) [](ancestors = 467635724)